init-localstack.sh

```
@@ -0,0 +1,2 @@
+#!/bin/bash
+awslocal s3 mb s3://${AWS_BUCKET_NAME:-team4you-files}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)